### PR TITLE
fix(dedicated.server): fix vrack attachment for legacy interfaces

### PR DESCRIPTION
**Pull request by marie-j**
Related : DTRSD-4300
Signed-off-by: Marie JONES <marie.jones@corp.ovh.com>

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicated/server/dedicated-server.routes.js [hunk 1]
```diff
@@ -70,6 +71,11 @@ angular.module('App').config(($stateProvider) => {
       serverName: /* @ngInject */ $transition$ => $transition$.params().productId,
       specifications: /* @ngInject */ (serverName, Server) => Server.getBandwidth(serverName),
       user: /* @ngInject */ currentUser => currentUser,
+      vrack: /* @ngInject */ (
+        Server,
+        serverName,
+      ) => Server.getVrack(serverName)
+        .catch(() => null),
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/dedicated/server/dedicated-server.routes.js]: No difference between each type of error ?

Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/dedicated/server/dedicated-server.routes.js]: We are stuck in an infinite loop 😅  (this question was already asked a lot this week, maybe if you do this intentionally, you could add a comment to say it's legit)

**Comment by antleblanc** [comment]
Hi @marie-j!

We recently merged #1336 which apply Prettier rules so could you please rebase
your branch on `develop`?

To easily resolve conflicts you can **ignore** `HEAD` changes and only keep yours.

After that you can run:

```sh
$ yarn run lint
```

This command will lint the entire codebase.

or:

Amend your commit to trigger the `pre-commit` hook (prettier will be launched
under the hood).

Thanks!

**Comment by JeremyDec** [comment]
@marie-j conflict

**Comment by marie-j** [comment]
I'm closing this as it has been blocked for a while now, I will reopen when unblocked

**Pull request closed without merging**